pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/LoadSheddingTask.java

```diff
@@ -52,25 +55,24 @@ public void run() {
         } catch (Exception e) {
```

Comment:
In the first line of method `run` method, we can add a check to make quick failure:

```java

public void run(){
    if (isCancel){
        return;
    }
    ...
}

```

Reply:
Thanks, good idea.